Sources/SystemMetrics/SystemMetrics.swift

```diff
@@ -272,8 +283,54 @@ public enum SystemMetrics {
             static let stimeTicks = 12
         }
 
+        func getSystemStartTimeInSecondsSinceEpoch() -> Int? {
```

Comment:
Instead of creating a function here and making the var optional we could just init inline above with 
```
var systemStartTimeInSecondsSinceEpoch: Int = {
    ....
}()
```

Reply:
Ah of course. My only question is that the inline closure would use some `CFile` methods, and that class is defined within the scope of the `linuxSystemMetrics` function. Are there any performance (or other) penalties if I move this `class` outside of the `func` (i.e. some missed compiler optimisation, etc)?

Reply:
Moving the class out of the method should be fine just make it `private`.